app/views/users/new.html.erb

```diff
@@ -0,0 +1,45 @@
+<!-- 
+<form action="/users" method="post" name="users" accept-charset="UTF-8">
+    <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>">
+
+    <label for="users[username]">Username:</label><br>
+    <input id="username" name="users[username]" type="text"/><br>
+
+    <label for="users[email]">Email:</label><br>
+    <input id="email" name="users[email]" type="email"/><br>
+
+    <label for="users[password]">Password:</label><br>
+    <input id="password" name="users[password]" type="password"/><br>
+
+    <input id="submit" type="submit" value="Create User"/>
+</form>
+-->
+<!-- 
+ <%= form_tag("/users", method: "post") do %>
```

Comment:
same as with the first form you should use the rails helpers `users_path` as the first parameter of the `form_tag()` helper